modules/scf/gen_config.sh

```diff
@@ -56,12 +58,27 @@ fi
 
 if [ "${SCF_OPERATOR}" == "true" ]; then
 
+
+if [ "$services" == ingress ]; then
+INGRESS_BLOCK="ingress:
```

Comment:
https://github.com/SUSE/catapult/pull/192#issuecomment-620717537, if ingress is not ready yet. Then we should exit here.

Reply:
Thing is, ingress configuration seems to be ready, I have compared it with tf3-c4.cap.suse.de ingress setup.

I can't get it to work on GKE. I get:
`Request error: Get https://api.ci.kubecf.charmedquarks.me: dial tcp 34.<edited>:443: connect: connection refused`

I get correct ingresses:
```
vic@viccuad ~/suse/catapult-aks/buildgke-lb$ kubectl describe ingress susecf-scf -n scf
Name:             susecf-scf
Namespace:        scf
Address:          34.<edited>
Default backend:  default-http-backend:80 (<none>)
TLS:
  susecf-scf-ingress-tls terminates *.ci.kubecf.charmedquarks.me,ci.kubecf.charmedquarks.me
Rules:
  Host                          Path  Backends
  ----                          ----  --------
  *.ci.kubecf.charmedquarks.me
                                /   router:443 (10.<edited>:443)
  ci.kubecf.charmedquarks.me
                                /   router:443 (10.<edited>:443)
Annotations:
  nginx.ingress.kubernetes.io/secure-backends:   true
  nginx.ingress.kubernetes.io/ssl-redirect:      false
  nginx.org/websocket-services:                  router
  kubernetes.io/ingress.class:                   nginx
  nginx.ingress.kubernetes.io/backend-protocol:  HTTPS
  nginx.ingress.kubernetes.io/proxy-body-size:   8m
Events:                                          <none>
```
And the router service seems to be ok:
```
NAME                    TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                                                                                            AGE                                            
router                  ClusterIP   None            <none>        80/TCP,443/TCP                                                                                     20m                                            
router-0                ClusterIP   10.<edited>   <none>        80/TCP,443/TCP                                                                                     20m                                                
```

I'm resolving the conversation, I believe the ingress code is ok.